src/ISM330DLCSensor.cpp

```diff
@@ -872,7 +872,7 @@ ISM330DLCStatusTypeDef ISM330DLCSensor::Set_G_FS(float fullScale)
   }
   else
   {
-    new_fs = ( fullScale <=  245.0f ) ? ISM330DLC_ACC_GYRO_FS_G_245dps
+    new_fs = ( fullScale <=  245.0f ) ? ISM330DLC_ACC_GYRO_FS_G_250dps
```

Comment:
we need also to modify the condition in this way: "new_fs = ( fullScale <= 250.0f ) ? ISM330DLC_ACC_GYRO_FS_G_250dps"